docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM gitlab.baltic-amadeus.lt:5005/progress/progress-docker/progress-build:12.6.0-centos
```

Comment:
I think we can use PROGRESS_BASE_IMAGE variable here

Reply:
Pretty sure we can't, .env is not directly accessible within the dockerfile. The approach probably would be to pass it as a build argument? Idk

Reply:
Something like that:
`FROM ${DB_NAME}`
and then add PROGRESS_BASE_IMAGE to environment variables in docker compose file